test/data/rules/bom

```diff
@@ -0,0 +1,21 @@
+﻿! model         = keycodes
+  my_model      = my_keycodes
+  *             = default_keycodes
```

Comment:
any chance we can put a comment in this file somewhere near the top to say that it starts with a bom? Otherwise it's a bit hard to tell, given the thing isn't supposed to render :)

Reply:
Good point! I added a comment and changed the file name to a more explicit one.